webanno/webanno
# #467  Ability to view other annotator's work in Annotation Page

Admin should be able to see others' work. When opening the OpenDocumentDialog on the AnnotationPage, it will list annotator user names and their documents.
After selecting the user name and document, it will show the version of the annotation of that particular annotator, in read-only mode.

**What's in the PR**
* This is only for the admin being able to see others' annotations in read-only mode.

**How to test manually**
* Go to the AnnotationPage as an Admin, you should be able to open others' documents but not edit them.
* Go to the annotationpage as a regular user, you should only be allowed to open (and edit) your own documents.

* Might also want to check that opening documents in INCEpTION still works.

**Documentation**
